### ISSUE TITLE
GraphQLfactory should allow for multiple schema files

### ISSUE THREAD

**Issue by TimothyMDean** [smoketurner/dropwizard-graphql#10]
I would like to use `dropwizard-graphql` for a project I'm working on but currently the schema is contained in multiple files. Doing this directly in graph-java I do the following:

```
TypeDefinitionRegistry registry = new TypeDefinitionRegistry();
registry.merge(parser.parse(new InputStreamReader(getClass().getClassLoader().getResourceAsStream(GRAPHQL_TYPES_PATH))));
registry.merge(parser.parse(new InputStreamReader(getClass().getClassLoader().getResourceAsStream(GRAPHQL_API_PATH))));
```

It appears that the current implementation of `GraphQLFactory` expects the schema to be in a single file that is set via a single method call.

It would be nice if dropwizard-graphql would allow me to spread my schema across multiple files in projects that want to do that.

**Comment by jplock** [comment]
Thanks for the suggestion @TimothyMDean, I'll see if I can work on this.

**Comment by jplock** [comment]
Submitted https://github.com/smoketurner/dropwizard-graphql/pull/11 to fix this.

**Comment by jplock** [comment]
Fixed in `v1.3.5-4`
